README.md

```diff
@@ -10,13 +10,13 @@ to provide a minimalist PSR-7 middleware framework for PHP, with the following
 features:
 
 - Routing. Choose your own router; we support:
-    - [Aura.Router](https://github.com/auraphp/Aura.Router)
+  - [Aura.Router](https://github.com/auraphp/Aura.Router)
     - [FastRoute](https://github.com/nikic/FastRoute)
     - [laminas-router](https://github.com/mezzio/mezzio-router)
 - DI Containers, via [PSR-11 Container](https://github.com/php-fig/container).
   Middleware matched via routing is retrieved from the composed container.
 - Optionally, templating. We support:
-    - [Plates](http://platesphp.com/)
+  - [Plates](http://platesphp.com/)
     - [Twig](http://twig.sensiolabs.org/)
     - [Laminas's PhpRenderer](https://github.com/laminas/laminas-view)
```

Comment:
These lists are all nested weird now - can you reduce the indent on all of the list items please?

Reply:
done